vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/tracing/SqlTracer.java

```diff
@@ -0,0 +1,94 @@
+package io.vertx.sqlclient.impl.tracing;
+
+import io.vertx.core.impl.ContextInternal;
+import io.vertx.core.spi.tracing.TagExtractor;
+import io.vertx.core.spi.tracing.VertxTracer;
+import io.vertx.sqlclient.SqlConnectOptions;
+import io.vertx.sqlclient.Tuple;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Function;
+
+/**
+ * Tracer for SQL client, wrapping the generic tracer.
+ */
+public class SqlTracer {
+
+  enum RequestTags {
+
+    // Generic
+    PEER_ADDRESS("peer.address", q -> q.tracer.address),
+    PEER_SERVICE("peer.service", q -> "todo"),
```

Comment:
`peer.service` is not used for now

Reply:
I think that peer.service is something the user can provide, I think we might simply have this part of the TracingOptions when creating Vert.x and leave it the tracer actually set it

Reply:
I think actually in our case we should not use this tag

Reply:
from https://docs.instana.io/ecosystem/opentracing/ "Set the peer.service tag if you don't know the remote side is instrumented. In general, it is not required as long as the remote side is monitored by Instana and, in that case, it will automatically have a service name set. If peer.service is set in the client call and the server is instrumented and has a custom service name, the results are undefined." so we don't need it as it's a backend not service

Reply:
+1 I think in our SQL client usecase we don't need to add this.